packages/cli/src/formatter.ts

```diff
@@ -523,6 +523,22 @@ export const formatConfiguredServices = (serviceNames: ReadonlyArray<string>): s
   return formattedServices.join('\n')
 }
 
+const formattedAdditionalServices = (services: ReadonlyArray<string>): string => {
```

Comment:
nit: be consistent with the other function names here
```suggestion
const formatAdditionalServices = (services: ReadonlyArray<string>): string => {
```